test/e2e/test_deb.py

```diff
@@ -108,4 +119,10 @@ def test_deb(container_with_installed_deb, tmpdir):
     run_command_on_container(container, "stress")
     run_command_on_container(container, "neofetch")
 
+    output = check_contains_file(container, '$HOME/hello.txt')
```

Comment:
Let's fix `check_contains_file` to return `run_command_on_container(container, command) == 'true`

https://github.com/tarantool/cartridge-cli/blob/master/test/utils.py#L1069